karoo_gp/base_class.py

```diff
@@ -367,7 +367,7 @@ def fx_data_load(self, filename):
 		
 		### PART 1 - load the associated data set, operators, operands, fitness type, and coefficients ###
 		# full_path = os.path.realpath(__file__); cwd = os.path.dirname(full_path) # for user Marco Cavaglia
-		cwd = os.getcwd()
+		cwd, _ = os.path.split(__file__)
```

Comment:
a) `cwd` is no longer the correct variable name, so we need to change this to `karoo_dir`

b)  use instead `karoo_dir = os.path.dirname(os.path.realpath(__file__))` per Ezio's suggestion

c) update all use-cases of `cwd` with `karoo_dir`